src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/ApiDataTypeDefs.java

```diff
@@ -73,7 +83,9 @@ public abstract class ApiDataTypeDefs {
       new PrimitiveApiDataTypeDef(
           ApiTypeName.TIMEUUID,
           DataTypes.TIMEUUID,
-          new ApiSupportDef.Support(false, true, true, true));
+          // Does not support counter as primitive column, list/set value or map key/value.
+          // TODO dod we want to relax this guardrail?
+          new ApiSupportDef.Support(false, ApiSupportDef.CollectionSupport.NONE, true, true, true));
```

Comment:
+1 for now, I am guessing this is because we are trying to reduce the ide of using timeuuid so it makes sense. May need to change later